packages/jsapi-shim/src/dh.types.ts

```diff
@@ -400,7 +400,7 @@ export interface Chart extends Evented {
   readonly title: string;
   readonly titleFont: string;
   readonly titleColor: string;
-  readonly isShowLegend: boolean;
+  readonly showLegend: boolean;
   readonly legendFont: string;
   readonly legendColor: string;
```

Comment:
Should we be passing these through too? Or are they passed through already

Reply:
They are not passed through; we're also not passing through `titleFont`. I've made a ticket for that: https://github.com/deephaven/web-client-ui/issues/985